ambuda/seed/texts/ramayana.py

```diff
@@ -7,6 +7,24 @@
 
 BASE_URL = "https://bombay.indology.info/ramayana/text/UD/Ram{n}.txt"
 
+TEI_HEADER = """<teiHeader xml:lang="en">
+  <fileDesc>
+    <titleStmt>
+        <title>Rāmāyaṇa</title>
+        <author>Vālmīki</author>
+    </titleStmt>
+
+    <publicationStmt>
+        <publisher>bombay.indology.info</publisher>
+        <availability>
+        <p>The electronic text of the Rāmāyaṇa is sourced from <a href="http://bombay.indology.info/mahabharata/statement.html">John Smith's website.</a></p>
```

Comment:
I believe this is from the Oriental Institute, Baroda https://archive.org/details/in.ernet.dli.2015.485534/page/n1/mode/2up